deploy/kubernetes/base/node.yaml

```diff
@@ -25,8 +30,9 @@ spec:
         - name: ebs-plugin
           securityContext:
             privileged: true
-          image: amazon/aws-ebs-csi-driver:latest
+          image: "amazon/aws-ebs-csi-driver:latest"
```

Comment:
NIT: redundant quota?

Reply:
I didn't add them to the helm chart and was trying to keep actual change to a minimum for this, but I will remove the quotes from the chart.